packages/core-ethereum/src/indexer/index.ts

```diff
@@ -279,15 +292,21 @@ class Indexer extends EventEmitter {
   }
 
   private async onChannelUpdated(event: Event<'ChannelUpdate'>): Promise<void> {
-    const channel = ChannelEntry.fromSCEvent(event)
+    let channel
+    try {
+      channel = await ChannelEntry.fromSCEvent(event, (a: Address) => this.getPublicKeyOf(a))
+    } catch (e) {
+      log('could not process channel event, skipping it', e)
```

Comment:
The error is thrown whenever we don't know a public key?

Reply:
indeed.